obp/dataset/synthetic_slate.py

```diff
@@ -1136,6 +1136,64 @@ def generate_evaluation_policy_pscore(
             )
         return pscore, pscore_item_position, pscore_cascade
 
+    def calc_evaluation_policy_action_dist(
+        self,
+        bandit_feedback: BanditFeedback,
+        evaluation_policy_logit_: np.ndarray,
+        is_factorizable: bool,
+    ):
+        """Calculate action distribution at each slot given evaluation policy logit.
+
+        Parameters
+        ----------
+        bandit_feedback: BanditFeedback
+            Logged bandit feedback data used for off-policy evaluation for the slate recommendation setting.
+
+        evaluation_policy_logit_: array-like, shape (n_rounds, n_unique_action)
+            Logit values of evaluation policy given context (:math:`x`), i.e., :math:`\\f: \\mathcal{X} \\rightarrow \\mathbb{R}^{\\mathcal{A}}`.
+
+        is_factorizable: bool, default=False
+            Whether the behavior and evaluation policies are factorizable or not.
+
+        Returns
+        ----------
+        evaluation_policy_action_dist: array-like, shape (n_rounds * len_list * n_unique_action, )
+            Action choice probabilities of evaluation policy for all possible actions
```

Comment:
[imo]

I think that :math:`\\pi_e(a_t(k) | x_t, a_t(1), \\ldots, a_t(k-1)) \\forall a_t(k) \\in \\mathcal{A}` does not mean `Action choice probabilities of evaluation policy for all possible actions`.
For instance,  :math:`\\pi_e(a_t(k) | x_t, a_t(1), \\ldots, a_t(k-1))` where :math:`a_t(1)` does not match `bandit_feedback["action"].reshape((n_rounds, len_list))[i][0]` (= the factual action of the 1st slot of round `i`)  is not included in the response of this function.

`Action choice probabilities of evaluation policy for all possible actions given the previous actions` would be a better explanation.


I'm sorry if I could not understand the implementation.

Reply:
I totally agree with you and will fix the docstrings. Thank you for the suggestion.